src/componentLibrary/Grid/Grid.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
```

Comment:
@crypto-engineer @peterslany I don't think we need stories for these components - documentation would be useful, but for a small dev team I think even that won't be necessary once we have these in use in a couple of places. Any thoughts?

Reply:
Morning @tomjeatt. I agree the stories are unnecessary for the layout components and documentation makes sense to me depending on how friendly we want to make our codebase to outer contributors (probably depends on interest of community to contribute too).

Reply:
Ta :) I'll leave the stories in until we've used the components in the app, and then I'll get rid of them. I'll add something to the backlog for docs - think we'll have time for this kind of thing once Rui starts.